wiki/Community/Discord_servers/Project_Loved_Taiko/en.md

```diff
@@ -0,0 +1,30 @@
+# Project Loved: Taiko Discord server
+
+::: Infobox
+
+<!-- lint ignore heading-increment -->
+
+#### Project Loved: Taiko Discord
+
+![Server banner](img/banner.jpg "Project Loved banner, designed by Tanza3D")
+
+|  |  |
+| :-- | :-- |
+| Invite link | <https://discord.gg/BuJ6TVPmtN> |
+
+:::
+
+**Project Loved: Taiko** is a community [Discord](https://discord.com) server designated for osu!taiko players and mappers alike looking for information about Project Loved or to discuss and suggest maps for consideration. Users are able to directly interact with the osu!taiko Loved Captains, other Project Loved Team members, and view Captain discussions publicly.
+
+## Staff
+
+| Role | User (Discord handle) |
+| :-- | :-- |
+| Server Owner | ::{ flag=CL }:: [-Kazu-](https://osu.ppy.sh/users/920861) (kiel_d01) |
+| **Captains** & Project Loved Team Members (Server Moderators) | ::{ flag=CL }:: [**-Kazu-**](https://osu.ppy.sh/users/920861) (kiel_d01), ::{ flag=GB }:: [**aceticke**](https://osu.ppy.sh/users/8838763) (aceticke), ::{ flag=US }:: [Blushing](https://osu.ppy.sh/users/5927823) (blooshing),    ::{ flag=NL }:: [**Boaz**](https://osu.ppy.sh/users/13302996) (13oaz), ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350) (gp_clayton), ::{ flag=JP }:: [**Eriha**](https://osu.ppy.sh/users/16320311) (4yuni), ::{ flag=EC }:: [**Gamelan4**](https://osu.ppy.sh/users/9856910) (gamelan4), ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350) (gp_clayton), ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) (hivie), ::{ flag=LT }:: [huu](https://osu.ppy.sh/users/6044237) (huuishuu), ::{ flag=GB }:: [**Horiiizon**](https://osu.ppy.sh/users/8071438) (horiiizon), ::{ flag=ES }:: [kadoen](https://osu.ppy.sh/users/12780575) (deranged_cultist), ::{ flag=MY }:: [**ler1211**](https://osu.ppy.sh/users/19901680) (ler1211), ::{ flag=SG }:: [megumic](https://osu.ppy.sh/users/7537133) (megumic),  ::{ flag=KR }:: [POCARI SWEAT](https://osu.ppy.sh/users/5082685) (poca_p0ca), ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) (radar), ::{ flag=BR }:: [**Sebola**](https://osu.ppy.sh/users/15821708) (sebola3461), ::{ flag=AT }:: [Yasuho](https://osu.ppy.sh/users/8458835) (yasuho) |
```

Comment:
this one's got extra spaces before boaz and my name duplicated, maybe some copy-paste error?

Reply:
yep will fix that